.gitignore

```diff
@@ -1,3 +0,0 @@
-
-.DS_Store
-.DS_Store
```

Comment:
If these changes to the `.gitignore` file aren't related in any way to string interoperability, I'd suggest omitting them in this PR.